federatedscope/db/processor/basic_processor.py

```diff
@@ -19,8 +19,8 @@ def get_table(self, table_name: str):
     def get_schema(self, table_name: str):
         return self.schemas[table_name]
```

Comment:
The variables `tables` and `schemas` should be kept by the `Accessor` class rather than `Processor`?